inter-operator/architecture/update.md

```diff
@@ -1,54 +0,0 @@
-# Upgrade Interoperator from the earlier releases(special handling, downtime if any)
```

Comment:
This is not put anywhere else. Should we keep this

Reply:
I am adding it with https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/829